websockets/unload-a-document/META.yml

```diff
@@ -0,0 +1,6 @@
+links:
+    - product: chrome
+      url: https://crbug.com/801564
```

Comment:
The bug title "Implement WebSockets over HTTP/2" makes it seem like a poor match, but the test was tweaked. I feel I'd need to dig deeper to know if it's a relevant bug or not. So let's link it, and unlink if a better one is filed. (Let's ignore that deleting links isn't supported in the UI.)